src/webgpu/api/validation/queue/destroyed/query_set.spec.ts

```diff
@@ -9,7 +9,107 @@ TODO: implement. (Search for other places some of these cases may have already b
 Consider whether these tests should be distributed throughout the suite, instead of centralized.
```

Comment:
Do you have an opinion about this after working with these tests? Feedback would be useful.

Reply:
These validations happen on queue submit, it is a good choice to put together the cases with same check points.